samples/bullet-samples/05_filtering_extended.py

```diff
@@ -0,0 +1,190 @@
+from panda3d.core import load_prc_file_data
+load_prc_file_data('', 'bullet-filter-algorithm groups-mask')
+
+import sys
+
+from direct.showbase.ShowBase import ShowBase
+from direct.showbase.InputStateGlobal import inputState
+
+from panda3d.core import AmbientLight
+from panda3d.core import DirectionalLight
+from panda3d.core import LVector3
+from panda3d.core import LVector4
+from panda3d.core import LPoint3
+from panda3d.core import TransformState
+from panda3d.core import BitMask32
+
+from panda3d.bullet import BulletWorld
+from panda3d.bullet import BulletPlaneShape
+from panda3d.bullet import BulletBoxShape
+from panda3d.bullet import BulletRigidBodyNode
+from panda3d.bullet import BulletDebugNode
+
+class Game(ShowBase):
+
+    def __init__(self):
+        ShowBase.__init__(self)
+        base.set_background_color(0.1, 0.1, 0.8, 1)
+        base.set_frame_rate_meter(True)
+
+        base.cam.set_pos(0, -20, 4)
+        base.cam.look_at(0, 0, 0)
+
+        # Light
+        alight = AmbientLight('ambientLight')
+        alight.set_color(LVector4(0.5, 0.5, 0.5, 1))

```

Comment:
We can just use a tuple to set the color, we don't need to use LVector4 for that.  We can remove the import to LVector4 too, then.